packages/@aws-cdk/aws-apprunner-alpha/lib/index.ts

```diff
@@ -1,3 +1,4 @@
 // AWS::AppRunner CloudFormation Resources:
 export * from './service';
 export * from './vpc-connector';
+export * from './observability-configuration';
```

Comment:
Can you please order this alphabetically?

Reply:
Thanks. I fixed it.